### PR TITLE
add documentation to the GetLanguage and GetLanguageCode reports

### PR DESCRIPTION
**Pull request by Findus23**
see https://forum.matomo.org/t/browser-language-language-code/38540

It seems like those two reports never got a documentation. If someone has an idea on how to improve the wording, just comment.

### CODE REVIEW

**Comment by tsteur** [comment]
@Findus23 could you update the failed system tests where the new documentation was added? We can otherwise also do it if you're busy

**Comment by Findus23** [comment]
@tsteur I waited with updating the tests in case someone has a better wording.

File: plugins/UserLanguage/lang/en.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
     "UserLanguage": {
         "BrowserLanguage": "Browser language",
         "LanguageCode": "Language code",
-        "PluginDescription": "Reports the language used by your visitors' browsers."
+        "PluginDescription": "Reports the language used by your visitors' browsers.",
+        "getLanguageDocumentation": "This report shows which Language the visitor's browsers are using. (e.g. \"English\")",
```
Comment (tsteur) [inline plugins/UserLanguage/lang/en.json:6]: should language be lower case @Findus23 (also below)? Looks otherwise good

**Comment by Findus23** [comment]
I think (but am not sure) that all tests have been fixed.

**Pull request merged**